prov/efa/src/rxr/rxr_init.c

```diff
@@ -265,8 +265,9 @@ static int rxr_info_to_rxr(uint32_t version, const struct fi_info *core_info,
 		}
 
 		/* Use a table for AV if the app has no strong requirement */
-		if (hints->domain_attr->av_type == FI_AV_UNSPEC)
-			info->domain_attr->av_type = FI_AV_TABLE;
+		if (hints->domain_attr)
+			if (hints->domain_attr->av_type == FI_AV_UNSPEC)
+				info->domain_attr->av_type = FI_AV_TABLE;
```

Comment:
Would this be better as:
```
if (!hints->domain_attr || hints->domain_attr->av_type == FI_AV_UNSPEC)
```
so you default to av table if no domain attributes are given?

Reply:
It would. We do default to table if nothing is given.